srcpkgs/imwheel/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'imwheel'
+pkgname=imwheel
+version=1.0.0pre12
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config libXmu libXext libXtst"
+makedepends="libX11-devel libXtst-devel libXmu-devel"
+short_desc="A universal mouse wheel and mouse stick translator for the X Windows System."
```

Comment:
s/A universal/Universal/

Reply:
Suggest `short_desc="Universial mouse wheel and stick translator for X11"`. That should be < 76 characters.

Reply:
> Suggest `short_desc="Universial mouse wheel and stick translator for X11"`. That should be < 76 characters.

OK, changed